Support/how-do-i-report-a-support-issue.md

```diff
@@ -101,3 +104,8 @@ When you reopen a ticket, it goes to the engineer that solved. Our engineers wor
 **How long do you wait before closing a ticket?**
 
 After a ticket has been Solved, the automation will close the the ticket after 3 days. All subsequent communication to the closed ticket will create a new ticket at Low priority.
+
+### *Chat requirements*
```

Comment:
Chat requirements should not be a major heading as its not part of the TOC or overall top level list of how you get support (email, phone, chat). a simple list of the requirements in the chat support area is better from a formatting perspective within the chat support section is recommended so its in line with standard TOC/headings